newIDE/app/src/Profile/LoginDialog.js

```diff
@@ -92,12 +85,7 @@ const LoginDialog = ({
       ]}
       cannotBeDismissed={loginInProgress}
       onRequestClose={onClose}
-      onApply={() => {
-        // This is a hack to avoid submitting the login form
-        // when submitting the forgot password form.
```

Comment:
just to check, is this not needed anymore?

Reply:
I checked and the bug seems to not be present anymore

Reply:
nice!